feat: adding printing statistics for networks (#19)
runziggurat/crunchy: README.md

  -V, --version                        Print version
```

> Running using command line prints also the parameters of the network before and IPS. Parameters are printed in the following format:

Reviewer: ```suggestion
The command line also prints the network parameters before and after applying the IPS algorithm output. Parameters are printed in the following format:
```

Author: 👍

```

> Statistics for final network:

Reviewer: ```suggestion
Statistics for the final network:
```

Author: 👍

----------------------------------------

> 

Reviewer: [opinion sharing]:
this may be a subjective thing, but I would find it more pleasing to my eyes without the newline here from the UI perspective

Author: 👍

Nodes count: 2472

Degree measures:
Average: 350.2698220064725
Median: 363
Min: 5, max: 472, delta: 467

Betweenness measures:
Average: 0.00020269986625605759
Median: 0.00021142414089966258
Min: 0.000000011402470586985047, max: 0.0005637115400558378, delta: 0.0005637001375852508

Closeness measures:
Average: 2.000959540368865
Median: 1.999216689819378
Min: 1.996533523649682, max: 2.995343149165006, delta: 0.9988096255153243

Eigenvector measures:
Average: 1.0000000000000013
Median: 1.0369942301865014
Min: 0.015083344270596788, max: 1.3408045956582166, delta: 1.3257212513876198
----------------------------------------

> ```

Reviewer: It would be nice to see the example of the delta printout in the example, even the initial network printout

Author: Added.
